pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -585,7 +594,13 @@ func getConditionCheckName(taskRunStatus map[string]*v1beta1.PipelineRunTaskRunS
 }
 
 // GetTaskRunName should return a unique name for a `TaskRun` if one has not already been defined, and the existing one otherwise.
-func GetTaskRunName(taskRunsStatus map[string]*v1beta1.PipelineRunTaskRunStatus, ptName, prName string) string {
+func GetTaskRunName(taskRunsStatus map[string]*v1beta1.PipelineRunTaskRunStatus, childRefs []v1beta1.ChildStatusReference, ptName, prName string) string {
```

Comment:
super nit: I think you can import and use constants for the `kind` of Run and TaskRun (here and elsewhere in the PR)

Reply:
I swear that I couldn't figure out where to find that to import. Maybe I missed something?

Reply:
they're [here](https://github.com/tektoncd/pipeline/blob/main/pkg/apis/pipeline/controller.go) but actually on second thought importing and using the controller name is a bit weird :/

Reply:
Oh right, I _did_ look at that and decided it felt too squirelly - I don't quite trust that the controller names are always going to be guaranteed to map one to one with the `kind`s. Feels like it might be worthwhile to add constants specifically for the `kind` name somewhere...